pkg/controller/controller_utils.go

```diff
@@ -124,9 +124,6 @@ func (r *RCExpectations) ExpectDeletions(rc *api.ReplicationController, dels int
 // Decrements the expectation counts of the given rc.
 func (r *RCExpectations) lowerExpectations(rc *api.ReplicationController, add, del int) {
 	if podExp, exists, err := r.GetExpectations(rc); err == nil && exists {
-		if podExp.add > 0 && podExp.del > 0 {
```

Comment:
this isn't worth the loadint. 
what can go wrong here, is go just being panicky? im ok with these being inaccurate as i'm just using them for logging. The memory isn't goint to get deallocated because these are members in a struct I have a ref to right?